app/res/xml/entity_select_search.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<searchable xmlns:android="http://schemas.android.com/apk/res/android"
+            android:label="@string/app_name"
+            android:hint="Search for case">
```

Comment:
Is this localizable? Also: We should avoid using the word 'case' on search stuff as the word has no meaning to our end users.